modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf

```diff
@@ -103,13 +97,24 @@
 			<c:choose>
 				<c:when test='<%= msvv.getFieldName() == "expirationDate" %>'>
 					<liferay-ui:message key="please-enter-a-valid-expiration-date-only-future-dates-are-accepted" />
+					<% isShowingExpirationError = true; %>
 				</c:when>
 				<c:otherwise>
 					<liferay-ui:message arguments="<%= HtmlUtil.escape(fieldLabelValue) %>" key="validation-failed-for-field-x" translateArguments="<%= false %>" />
 				</c:otherwise>
 			</c:choose>
 		</liferay-ui:error>
 
+		<c:if test="<%= !isShowingExpirationError %>">
+			<c:if test="<%= ddmFormAdminDisplayContext.isFormExpired(ddmFormAdminDisplayContext.getDDMFormInstance()) %>">
+				<clay:alert
```

Comment:
I remember I asked you to bring this warning closer to the related error, but, now that I think about it, if we have more than one exception happening on the page, this warning could be displayed in the middle of errors. Like:
Error 1
Error 2
Warning
Error 3

Perhaps, we should put this whole block at the bottom of the file, after the last error, so errors would have priority. What do you think? If you want, we can talk to Julia about it. It's your call 😃

Reply:
I think that this is a good solution, but we can check with her just to make sure it's according to the expecting design. Thanks for the review!!